shared/chat/search-header.js

```diff
@@ -91,6 +92,7 @@ export default compose(
       },
       onSelectService: (props: OwnProps & {_onSelectService: Function}) => nextService => {
         props._onSelectService(nextService)
+        props.clearSearchResults()
```

Comment:
this is fine to get this out the door but, i think longer term we should not expose this to the components. clearSearchResults and .search should both be a side effect of the changing of the service, so the saga really should handle this. we basically never want these things to be separated so we shouldn't trust the callers to do this. This'll lead to bugs where these behaviors will get disconnected somehow